conf/machine/juno.conf

```diff
@@ -25,6 +25,10 @@ OPTEEOUTPUTMACHINE = "vexpress"
 
 SERIAL_CONSOLE = "115200 ttyAMA0"
 
+# For qemu arm64
+IMAGE_FSTYPES_append = " ext4.gz"
```

Comment:
nit.. either you use a space at the end or not, but please don't do both. looking at this layer it looks like it's generally only before, and not after.